### PR TITLE
[FC-0049] feat: Update permissions to remove object tags

### PR DESCRIPTION
**Pull request by ChrisChV**
## Description

This updates rules to add `can_remove_object_tag_objectid` to Objet tag views

## Supporting information

- Github issue: https://github.com/openedx/modular-learning/issues/204
- Internal ticket: [FAL-3720](https://tasks.opencraft.com/browse/FAL-3720)

## Testing instructions

- Follow the testing instructions in https://github.com/openedx/frontend-app-course-authoring/pull/987

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ChrisChV! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: openedx_tagging/core/tagging/rules.py [hunk 1]
```diff
@@ -99,6 +99,16 @@ def can_view_object_tag_objectid(_user: UserType, _object_id: str) -> bool:
...
 
 
+@rules.predicate
+def can_remove_object_tag_objectid(_user: UserType, _object_id: str) -> bool:
+    """
+    Everybody can remove object tags from any objects.
+
+    This rule could be defined in other apps for proper permission checking.
+    """
+    return True
```
Comment (bradenmacdonald) [inline openedx_tagging/core/tagging/rules.py:109]: Can you move this down below `can_change_object_tag_objectid` ? Also the default value shouldn't be True - the default value should be the same as `can_change_object_tag_objectid`, so just call that function and return the same value.

**Review by bradenmacdonald — CHANGES_REQUESTED** [review]

File: openedx_tagging/core/tagging/rules.py (hunk 1 above)
Comment (ChrisChV) [reply openedx_tagging/core/tagging/rules.py]: @bradenmacdonald Updated here https://github.com/openedx/openedx-learning/pull/188/commits/bf49f694ece7c1bcc0fc50b8919111ecb1b158eb

File: tests/openedx_tagging/core/tagging/test_views.py [hunk 2]
```diff
@@ -874,7 +874,8 @@ def test_retrieve_object_tags_query_count(self):
         Test how many queries are used when retrieving object tags and permissions
         """
         expected_perm = True
-        object_id, sort_test_applied_result = self.prepare_for_sort_test(expected_perm)
+        expoected_remove_perm = False
+        object_id, sort_test_applied_result = self.prepare_for_sort_test(expoected_remove_perm)
```
Comment (bradenmacdonald) [inline tests/openedx_tagging/core/tagging/test_views.py:878]: Minor typo: `expoected_remove_perm`

**Review by bradenmacdonald — APPROVED** [review]

File: tests/openedx_tagging/core/tagging/test_views.py (hunk 2 above)
Comment (ChrisChV) [reply tests/openedx_tagging/core/tagging/test_views.py]: Fixed: https://github.com/openedx/openedx-learning/pull/188/commits/0df63a1e8d7d6fe15bc8c824b3519d5d09379f4b

**Comment by openedx-webhooks** [comment]
@ChrisChV 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-learning&entry.867055334=https://github.com/openedx/openedx-learning/pull/188&entry.1484655318=https://github.com/ChrisChV&entry.752974735=2024-05-07+20:19&entry.1917517419=2024-05-08+19:14&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**